app/Http/Controllers/Api/V1/SearchController.php

```diff
@@ -0,0 +1,224 @@
+use Illuminate\Http\Response;
+use App\Repositories\Activity\ActivityRepositoryInterface;
+use Illuminate\Support\Facades\Validator;
+
+/**
+ * @authenticated
+ * 
+ * @group  Search API
+ *
+ * APIs for search
+ */
+class SearchController extends Controller
+{
+    private $activityRepository;
+
+    /**
+     * SearchController constructor.
+     *
+     * @param ActivityRepositoryInterface $activityRepository
+     */
+    public function __construct(ActivityRepositoryInterface $activityRepository)
+    {
+        $this->activityRepository = $activityRepository;
+    }
+
+    /**
+     * Deep Linking Search
+     *
+     * Search projects, playlists and activities for deep linking
+     * 
+     * @queryParam  query required Query to search. Example: test
+     * @queryParam  sort Field to sort by. Example: created_at
+     * @queryParam  order Order to sort by. Example: desc
+     * @queryParam  from Index where the pagination start from. Example: 0
+     * @queryParam  size Number of records to return. Example: 10
+     * 
+     * @response  {
+     *     "projects":{
+     *        "457":{
+     *           "id":457,
+     *           "name":"Text Structure Lesson 2 Problem and Solution",
+     *           "description":"Learning Objective\nThe learner will define and describe the main characteristics...",
+     *           "thumb_url":"/storage/uploads/3zjZuLoQrRk0MZ5wP7UPyOut5zUybf3tW3a4Q2M1.png",
+     *           "mongo_userid":"5ef5300e41668b53ea5ed1b3",
+     *           "starter_project":false,
+     *           "created_at":"2020-07-17T17:49:01.000000Z",
+     *           "updated_at":"2020-08-06T13:28:07.000000Z",
+     *           "deleted_at":null,
+     *           "playlists":{
+     *              "225":{
+     *                 "id":225,
+     *                 "title":"Solving Ratio and Rate Problems",
+     *                 "project_id":64,
+     *                 "order":null,
+     *                 "mongo_projectid":"5f3ae92a924a1d5ddf44dd80",
+     *                 "created_at":null,
+     *                 "updated_at":null,
+     *                 "deleted_at":null,
+     *                 "activities":{
+     *                    "993":{
+     *                       "id":993,
+     *                       "playlist_id":225,
+     *                       "title":"",
+     *                       "type":"h5p",
+     *                       "content":"",
+     *                       "h5p_content_id":17474,
+     *                       "thumb_url":"/storage/uploads/5f3aedeba24fb.jpeg",
+     *                       "subject_id":null,
+     *                       "education_level_id":null,
+     *                       "shared":false,
+     *                       "order":11,
+     *                       "mongo_playlistid":"5f3aeccd924a1d5ddf44ddd0",
+     *                       "created_at":null,
+     *                       "updated_at":null,
+     *                       "deleted_at":null
```

Comment:
Please add space after : for each field.

Reply:
Added space after : for each field.